index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>microverse_trails_with_milenitic</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="body">
+        <div id="body-wrapper">
+    <div id="main">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </div>
+    <div id="form">
+        <form id="survey-form">
+            <label>Name</label>
+            <input type="text" id="name" placeholder="Enter your name" />
+            <label>Email</label>
+            <input type="email" id="eamil" placeholder="Enter your email">
+            <label>Age</label>
+            <input type="number" id="number" placeholder="Age">
```

Comment:
@Santosh-Konappanavar :wave: 

- [ ] Kindly be clear while you are suggesting so that students can understand easily. 
  - You suggest, `I think It would be better to always use the label tag for attributes`.
  - I  think in this comment you wanted to suggest like that, `I think it would be better to always use the for attributes in label tag`.

- [ ] Kindly use [Grammarly](https://app.grammarly.com/) so that there is no spelling mistake and no capital letters in the middle of any sentence.